src/main/java/org/sagebionetworks/bridge/services/ParticipantService.java

```diff
@@ -866,6 +880,10 @@ public StudyParticipant updateIdentifiers(Study study, CriteriaContext context,
             sendEmailVerification = true;
             accountUpdated = true;
         }
+        if (update.getSynapseUserIdUpdate() != null && account.getSynapseUserId() == null) {
+            account.setSynapseUserId(update.getSynapseUserIdUpdate());
+            accountUpdated = true;
+        }
```

Comment:
Should there be a unit test for updating Synapse ID via updateIdentifiers()?

Reply:
Added